src/DynamoCore/Graph/Nodes/PortModel.cs

```diff
@@ -282,24 +263,34 @@ public bool ShouldKeepListStructure
             }
         }
 
+        /// <summary>
+        /// Returns true if the port has connectors or if the 
+        /// default value is enabled and not null. Otherwise, returns false.
+        /// </summary>
+        internal bool IsConnected
```

Comment:
API Change: `PortModel.IsConnected` property is added. This property replaces functionality previously contained in the `NodeModel.HasInput` and `NodeModel.HasConnectedInput` properties.